src/hub.js

```diff
@@ -317,8 +319,19 @@ async function updateEnvironmentForHub(hub) {
     THREE.Cache.clear();
 
     const onLoadingEnvironmentReady = () => {
-      environmentEl.setAttribute("gltf-model-plus", { src: sceneUrl });
       environmentEl.removeEventListener("model-loaded", onLoadingEnvironmentReady);
+
+      if (scene.is("entered")) {
+        // We've already entered, so move to new spawn point
```

Comment:
a somewhat nicer way to do this would be to fire an event, but then i'll need to revisit the way the event handling is happening in `spawn-controller`, which seemed like a potential rathole